dspace/config/clarin-dspace.cfg

```diff
@@ -144,3 +144,12 @@ matomo.site.id = 1
 matomo.tracker.bitstream.site_id = 1
 matomo.tracker.oai.site_id = 1
 statistics.cache-server.uri = http://cache-server.none
+
+##### Citacepro config #####
+rest.properties.exposed= citace.pro.url
+citace.pro.url = https://www.citacepro.com/api/dspace/citace/oai
+citace.pro.url = dspace.tul.cz
```

Comment:
I think it will be better if we don't have two properties with the same name and with different info. For the value e.g., dspace.tul.cz please change the property name to `citace.pro.university`